src/kern/npf_inet.c

```diff
@@ -425,6 +425,7 @@ npf_cache_ip(npf_cache_t *npc, nbuf_t *nbuf)
 {
 	const void *nptr = nbuf_dataptr(nbuf);
 	const uint8_t ver = *(const uint8_t *)nptr;
+	npf_t *npf = npf_getkernctx();
```

Comment:
@m00nbsd: Can you please replace this line with the following:
```c
const npf_t *npf = npc->npc_ctx;
```
Note: conceptually, there can be multiple NPF instances; `npf_getkernctx()` should generally be applicable only for npf_os.c and maybe kernel module hooks.